README.md

```diff
@@ -28,27 +28,27 @@ See ["Caching dependencies to speed up workflows"](https://docs.github.com/en/ac
 * Fixed the download stuck problem by introducing a timeout of 1 hour for cache downloads.
 * Fix zstd not working for windows on gnu tar in issues.
 * Allowing users to provide a custom timeout as input for aborting download of a cache segment using an environment variable `SEGMENT_DOWNLOAD_TIMEOUT_MINS`. Default is 60 minutes.
-* Two new actions available for granular control over caches - [restore](restore/action.yml) and [save](save/action.yml)
+* New actions are available for granular control over caches - [restore](restore/action.yml) and [save](save/action.yml).
 * Support cross-os caching as an opt-in feature. See [Cross OS caching](./tips-and-workarounds.md#cross-os-cache) for more info.
 
-Refer [here](https://github.com/actions/cache/blob/v2/README.md) for previous versions
+See the [v2 README.md](https://github.com/actions/cache/blob/v2/README.md) for older updates.
 
 ## Usage
 
 ### Pre-requisites
 
-Create a workflow `.yml` file in your repositories `.github/workflows` directory. An [example workflow](#example-cache-workflow) is available below. For more information, reference the GitHub Help Documentation for [Creating a workflow file](https://help.github.com/en/articles/configuring-a-workflow#creating-a-workflow-file).
+Create a workflow `.yml` file in your repository's `.github/workflows` directory. An [example workflow](#example-cache-workflow) is available below. For more information, see the GitHub Help Documentation for [Creating a workflow file](https://help.github.com/en/articles/configuring-a-workflow#creating-a-workflow-file).
 
-If you are using this inside a container, a POSIX-compliant `tar` needs to be included and accessible in the execution path.
+If you are using this inside a container, a POSIX-compliant `tar` needs to be included and accessible from the execution path.
 
 If you are using a `self-hosted` Windows runner, `GNU tar` and `zstd` are required for [Cross-OS caching](https://github.com/actions/cache/blob/main/tips-and-workarounds.md#cross-os-cache) to work. They are also recommended to be installed in general so the performance is on par with `hosted` Windows runners.
 
 ### Inputs
 
+* `key` - An explicit key identifying a cache. See [creating a cache key](#creating-a-cache-key).
```

Comment:
Technically a key does not identify a cache uniquely which I feel this sentence seems to be indicating. Hence better to go back to older text.

```suggestion
* `key` - An explicit key for restoring and saving the cache. See [creating a cache key](#creating-a-cache-key).
```

Reply:
Would you accept `naming` instead of `identifying`?

I'm trying to:
* avoid the awkwardness of `restoring and saving` instead of `saving and restoring` -- yes, I get that the sequence is restore and then save (although I believe that in practice it will really only restore or save, because if the cache exists to restore, it will not save since caches are write-once by id)
* harmonize the text across all three files

Reply:
I've applied this change:
```diff
-* `key` - An explicit key identifying a cache. See [creating a cache key](../README.md#creating-a-cache-key).
+* `key` - An explicit key for a cache entry. See [creating a cache key](../README.md#creating-a-cache-key).
```

Which seems like a more correct entry. It's consistent with the wording in nearby lines, avoids the problem of the restore/save, and avoids using the awkward `identifying`.